prepare.yml

```diff
@@ -186,7 +186,7 @@
       tags: ndt-e2e
       environment:
         http_proxy: "{{ http_proxy }}"
-      raw: "git clone {{ ndt_e2e_client_git }} {{ ndt_e2e_client_dir }}"
+      raw: "git clone --branch master {{ ndt_e2e_client_git }} {{ ndt_e2e_client_dir }}"
```

Comment:
I think that should never be the case for us, but for whatever reason I was getting git errors about not being able to resolve "HEAD". We always mean master here so it seemed reasonable to me to add that explicitly.